### PR TITLE
Issue 1112 ec sei params

### LINKED ISSUE #1112: Some parameters not getting set by EC SEI submodel
The EC SEI submodel doesn't set things like loss of lithium

### PR DESCRIPTION
**Pull request by rtimms**
# Description
 - Adds new variables used to track the total lithium concentration (thanks @ferranbrosa !)
 - Fix a bug where not all parameters are set by the EC SEI model (e.g. loss of lithium)
 - Slightly reformat the way variables related to concentration are set by the sei models  

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #1112  and #1117 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/master/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: pybamm/models/submodels/electrolyte_diffusion/base_electrolyte_diffusion.py [hunk 1]
```diff
@@ -105,6 +105,37 @@ def _get_standard_flux_variables(self, N_e):
...
+        Returns
+        -------
+        variables : dict
+            The variables which can be derived from the flux in the
+            electrolyte.
+        """
+
+        c_e_typ = self.param.c_e_typ
+        L_x = self.param.L_x
+
```
Comment (rtimms) [inline pybamm/models/submodels/electrolyte_diffusion/base_electrolyte_diffusion.py:128]: should this be epsilon * c_e here?

Comment (tinosulzer) [reply pybamm/models/submodels/electrolyte_diffusion/base_electrolyte_diffusion.py:128]: Yep

File: pybamm/models/submodels/interface/sei/ec_reaction_limited.py [hunk 2]
```diff
@@ -131,3 +111,48 @@ def set_initial_conditions(self, variables):
         j_sei_0 = pybamm.Scalar(0)
 
         self.initial_conditions = {L_sei: L_sei_0, j_sei: j_sei_0}
+
+    def _get_standard_concentraion_variables(self, variables):
```
Comment (tinosulzer) [inline pybamm/models/submodels/interface/sei/ec_reaction_limited.py:115]: If this function is only ever called from `get_coupled_variables`, there is no need to make it a separate function. Ideally, we should find a way to make it use the method from `base_sei` (e.g. with inner set to 0 and outer representing the total)

**Review by tinosulzer — COMMENTED** [review]
Looks good, but I think there should be a better way to do #1112 .

Also, there are some `.pkl` files in `examples/notebooks` to remove. They should probably be removed when running the notebooks (using `os.remove`)

**Review by tinosulzer — APPROVED** [review]
Thanks @rtimms , merging

**Pull request merged**